readthedocs/api/v2/views/integrations.py

```diff
@@ -249,14 +253,14 @@ def get_external_version_response(self, project):
             "versions": [to_build] if to_build else [],
         }
 
-    def get_deactivated_external_version_response(self, project):
+    def get_closed_external_version_response(self, project):
         """
-        Deactivate the external version on merge/close events and return the API response.
+        Close the external version on merge/close events and return the API response.
 
         Return a JSON response with the following::
 
             {
-                "version_deactivated": true,
+                "closed": true,
```

Comment:
This is changing the API response. Do we have any concerns about this? In theory, nobody should be listen to this response, tho. We reply this to GitHub / GitLab.

Reply:
Yea, should be fine.